doc/man1/openssl-verify.pod.in

```diff
@@ -314,6 +109,23 @@ with a B<-->.
 
 {- $OpenSSL::safe::opt_trust_item -}
 
+{- $OpenSSL::safe::opt_v_item -}
+
+This command ignores many errors, in order to allow all the problems with a
```

Comment:
It isn't that errors are "ignored", rather verification continues after reporting the error, the final outcome is still verification failure if any of these errors are encountered. Frankly, we should probably continue for all errors, not just the special-cased ones.  I can't think of a good reason to immediately abort verification in the other error cases...

Reply:
The end result is that multiple errors are reported if present, but only up to the first error that is not one of the special ones. Again, not sure why...

Reply:
I think "ignores errors" is correct since the function sets "ok=1"  Is that wrong?

I agree the behavior is kinda half-way and should be fixed in another PR.  I'm just trying to improve the documentation in this PR.

Reply:
Setting `ok = 1` just means **continue**, it does not clear the error. Ultimately `sctx->error` is still set.

Reply:
So maybe just: "If a certificate chain has multiple problems, this program tries to display all of them." And remove the specific list of errors?

Reply:
Perhaps so. And I'll open an issue about whether it should continue on all errors (if I don't forget, and nobody else beats me to it).

Reply:
Beat you to it: #10750 :)